tests/integration/test_backup_restore_azure_blob_storage/configs/users.xml

```diff
@@ -0,0 +1,8 @@
+<clickhouse>
```

Comment:
This `users.xml` doesn't seem to be necessary.

Reply:
addressed it